spec/models/piece_spec.rb

```diff
@@ -1,6 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe "#is_obstructed" do
+    context "when moving horizontally" do
+      it "returns true if something is in the way" do
+        rook = Rook.create(x_cord: 0, y_cord: 0, game_id: 1)
+        pawn = Pawn.create(x_cord: 1, y_cord: 0, game_id: 1)
+        expect(rook.is_obstructed?(5,0)).to eq true
+      end
+
+      it "returns false if path clear and there is no obstruction" do
+        rook = Rook.create(x_cord: 3, y_cord: 3, game_id: 1)
```

Comment:
@ilyakrasnov the things I've seen these tests return are: nil, opposite of what it should return, or method returns all true or all false (so that at least one of the tets pass -_-//), or a number. 

I think you will be able to throw me a quick hint and give me a new shiny path to follow.  I've been staring at this for 4 evenings now and am in a rut that I can't steer out of.  @catherinezzy has been working on it, too.  The logic seems like it should work, but writing same code over and over with failing tests is proving our syntax isn't working out.... Thank you!